keyboards/bpiphany/ghost_squid/keymaps/default/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
This file can be deleted.

Reply:
I added it, because I noticed that `frosty_flake` includes such file; removed.